thirdPartyCode/three/TransformControls.js

```diff
@@ -21,6 +21,7 @@ import {
 } from './three.module.js';
 
 const _raycaster = new Raycaster();
+_raycaster.layers.set(10);
```

Comment:
Can you add a comment at or near the top of this file explaining that you've modified this file and why you've modified this file, so that in future someone doesn't accidentally just swap out the file with an updated version from the three.js library (if they have updates in the future)